sample/__init__.py

```diff
@@ -1,4 +1,29 @@
+import pkg_resources
+from os import path
+from setuptools.config import read_configuration
+
+
+def _extract_version(package_name):
+    try:
```

Comment:
This is code that we're recommending everyone include in their projects - I don't think we want an `_extract_version` like this, or `pkg_resources` and `setuptools` dependencies, in such a recommendation.

The existing code doesn't offer a "get the package version" function, or try to address how the application should expose its version at runtime. I'd rather that remain the case, but even if we were to decide that we do that, it should be a separate PR (again!)